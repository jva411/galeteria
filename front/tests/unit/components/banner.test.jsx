/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import Banner from '../../../components/banner'
import { render, screen, cleanup } from '@testing-library/react'

jest.mock('next/router', () => ({
    useRouter() {
        return { pathname: '/'};
    },
}));

afterEach(cleanup)

describe('unit tests of component Banner', () => {
    test('should render text', () => {
        render(<Banner />)

        expect(screen.getByText(/Lenovo Internal Use Only/iu)).toBeInTheDocument()
    })

    test('should Banner match with Snapshot', () => {
        const tree = renderer.create(<Banner />).toJSON()

        expect(tree).toMatchSnapshot()
    })
})
