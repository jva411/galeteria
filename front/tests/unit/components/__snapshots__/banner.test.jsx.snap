// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit tests of component Banner should Banner match with Snapshot 1`] = `
<div
  className="css-1f132d2"
>
  <p
    className="chakra-text css-w8zzbh"
  >
    Lenovo Internal Use Only
  </p>
</div>
`;
